cms/djangoapps/contentstore/utils.py

```diff
@@ -2385,7 +2385,7 @@ def get_xblock_render_context(request, block):
     return ""
 
 
-def _create_or_update_component_link(course_key: CourseKey, created: datetime | None, xblock):
+def _create_or_update_component_link(created: datetime | None, xblock):
```

Comment:
The changes in this file are not related to the PR. We can get the `course_key` from `xblock.usage_key.course_key`, so we don't need it as a parameter here.